newIDE/app/public/index.html

```diff
@@ -61,5 +61,24 @@
         </p>
       </div>
     </noscript>
+    <script>
+      (function() {
+        // Apply background color for startup
+        var backgroundColors = {
+          Dark: '#252525',
+          Nord: '#2E3440',
+          'Solarized Dark': '#00212B'
+        };
+        var themeName = JSON.parse(localStorage.getItem("gd-preferences"))['themeName'];
```

Comment:
Add a try/catch around the parsing - if for some reason the preferences are corrupted, we should not crash.